specification/metrics/quantityinpricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Quantity in Pricing Unit
+
+The Quantity in Pricing Unit represents the volume of a given resource and/or service used or purchased based on the [Pricing Unit](#pricingunit). Distinct from [Quantity In Usage Unit](#quantityinusageunit) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
+
+The QuantityInPricingUnit column MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeType](#chargetype) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying QuantityInPricingUnit by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
style "guide" (? not sure if this is more just a convention)

When referring to the column, just refer to it as its ID, without additional 'the' or 'column'.

Open to discussion on it, but have seen varying usage within these PRs.

```suggestion
QuantityInPricingUnit MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeType](#chargetype) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying QuantityInPricingUnit by a unit price MUST produce a result equal to the corresponding cost metric.
```

Reply:
There's a few of these that we will scrub at the end across all.